src/spec/function_prototypes.hpp

```
@@ -2,9 +2,13 @@
 // SPDX-License-Identifier: MIT
 #pragma once
 
+#include <cstdint>
+
 #include "spec/ebpf_base.h"
 
 namespace prevail {
+
+constexpr uint64_t map_type_bit(uint32_t type) { return uint64_t{1} << type; }
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)